recipes/livekraken/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "livekraken" %}
+{% set version = "1.0" %}
+{% set sha256 = "f1cb0fcceb86a31cf5192eb5bebb85149bbaf1899cc80d5b1074f520f1c257d5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://gitlab.com/SimonHTausch/LiveKraken/uploads/be6dfaaebf310a56d7cdc9676c5764b2/LiveKraken-master.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  has_prefix_files:
+    - libexec/livekraken
+    - libexec/livekraken-build
+    - libexec/livekraken-filter
+    - libexec/livekraken-mpa-report
+    - libexec/livekraken-report
+    - libexec/livekraken-translate
+
+requirements:
+  build:
+    - perl
+    - gcc
+    - boost 
+
+  run:
+    - jellyfish 1.*
+    - perl
+    - libgcc
+
+test:
+  commands:
+    - livekraken --version
+    - livekraken-build --version
+    - livekraken-filter --version
+    - livekraken-mpa-report --version
+    - livekraken-report --version
+    - livekraken-translate --version
+
```

Comment:
There are Python scripts in the livekraken repo, and one is documented in the README, but they don't appear to be installed here, is that intentional?